vnext/local-cli/config/configUtils.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+// @ts-check
+
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+
+const xmldoc = require('xmldoc');
+
+// find VS files matching the pattern
```

Comment:
Nit: JS tends to use JSDoc with has Javadoc style "/**". Vs code will auto close it for you.

Reply:
Fixed.